src/Repository/CommitteeRepository.php

```diff
@@ -334,4 +337,32 @@ public function paginate(int $offset = 0, int $limit = SearchParametersFilter::D
 
         return new Paginator($query);
     }
+
+    public function retrieveTopCommitteesInReferentManagedArea(Adherent $referent, $mostActive = true): array
```

Comment:
The semantic does not look very clear if you just check the method name and parameters.

What does `Top` mean ? 5 ? 10 ? 15 ? Perhaps it should be a parameter of the method ?

If `$mostActive` is `false` it means less active ? Not that clear just by reading this line. A constant would make a lot of sense as an alternative (`Enum` class can be used for example).

Reply:
@B-Galati It's a good idea to add a parameter to limit the number of top committees, but it seems to me overkill to use `Enum` class just for result ordering. And I doubt that it will be useful somewhere else in the project.

Reply:
`Enum` is just an option. You can use anything you want as long as it's clear.

Another solution is to have 2 differents method name 

https://martinfowler.com/bliki/FlagArgument.html
http://www.nichesoftware.co.nz/2016/05/15/code-gardening-bools-are-evil.html